cookbook/security/remember_me.rst

```diff
@@ -48,14 +43,62 @@ are shown here:
                 'main' => array(
                     'remember_me' => array(
                         'key'      => '%secret%',
-                        'lifetime' => 31536000, // 365 days in seconds
+                        'lifetime' => 604800, // 1 week in seconds
                         'path'     => '/',
-                        'domain'   => '', // Defaults to the current domain from $_SERVER
                     ),
                 ),
             ),
         ));
 
+The ``remember_me`` firewall defines the following configuration options:
+
+``key``
+    (default value: ``null``) The value used to encrypt the cookie's content.
```

Comment:
Very nice suggestion! Indeed that would be much better. Done here: 63890b1 Thanks!